.happy/terraform/modules/ecs-stack/main.tf

```diff
@@ -141,10 +144,14 @@ module backend_service {
   security_groups            = local.security_groups
   task_role_arn              = local.ecs_role_arn
   service_port               = 5000
+
+  # 100gb of disk storage allocated for the task running backend container 
+  task_storage_size_gb       = 100
   memory                     = var.backend_memory
   cpu                        = var.backend_cpus * 1024
   cmd                        = local.backend_cmd
   deployment_stage           = local.deployment_stage
+  wmg_disk_cache_path        = "/tmp/wmg_disk_cache"
```

Comment:
I'm not sure this is relevant in Docker, but Linux generally has cleanup processes that can remove files in `/tmp/`. I suggest another location (especially since the cache here isn't really temporary).

Reply:
@ebezzi - will fix